ui/pages/confirmations/components/confirm/header/header-info.tsx

```diff
@@ -68,6 +77,11 @@ const HeaderInfo = () => {
     });
   }
 
+  const isShowAdvancedDetailsToggle: boolean = Boolean(
+    currentConfirmation?.type &&
```

Comment:
Minor, but we shouldn't need this check and the `Boolean` cast if we just cast inside the includes such as:

`const isShowAdvancedDetailsToggle = REDESIGN_TRANSACTION_TYPES.includes(currentConfirmation?.type as TransactionType);`